cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/DefaultKafkaQuotasManager.java

```
@@ -0,0 +1,226 @@
+     */
+    public static Future<Void> reconcileDefaultUserQuotas(
+        Reconciliation reconciliation,
+        Vertx vertx,
+        AdminClientProvider adminClientProvider,
+        PemTrustSet pemTrustSet,
+        PemAuthIdentity pemAuthIdentity,
+        QuotasPlugin quotasPlugin
+    ) {
+        Promise<Void> promise = Promise.promise();
+        LOGGER.debugCr(reconciliation, "Reconciling default user quotas in Kafka");
+        String bootstrapHostname = KafkaResources.bootstrapServiceName(reconciliation.name()) + "." + reconciliation.namespace() + ".svc:" + KafkaCluster.REPLICATION_PORT;
+
+        LOGGER.debugCr(reconciliation, "Creating AdminClient for setting default quota using {}", bootstrapHostname);
+        Admin kafkaAdmin = adminClientProvider.createAdminClient(bootstrapHostname, pemTrustSet, pemAuthIdentity);
+
+        boolean isNotKafkaPlugin = !(quotasPlugin instanceof QuotasPluginKafka);
+        QuotasPluginKafka quotasPluginKafka = isNotKafkaPlugin ? emptyQuotasPluginKafka() : (QuotasPluginKafka) quotasPlugin;
+
+        List<ClientQuotaAlteration.Op> ops = prepareQuotaConfigurationRequest(quotasPluginKafka);
+
+        return shouldAlterDefaultQuotasConfig(reconciliation, vertx, kafkaAdmin, ops, isNotKafkaPlugin)
+            .compose(shouldUpdateQuotas -> {
+                if (shouldUpdateQuotas) {
+                    ClientQuotaAlteration clientQuotaAlteration = new ClientQuotaAlteration(DEFAULT_USER_ENTITY, ops);
+                    alterQuotas(reconciliation, vertx, kafkaAdmin, clientQuotaAlteration)
+                        .onComplete(result -> {
+                            if (result.succeeded()) {
+                                LOGGER.infoCr(reconciliation, "Successfully altered default user quotas");
+                            } else {
+                                LOGGER.warnCr(reconciliation, "Failed to alter default user quotas", result.cause());
```

I still don't understand why are you ignoring this error.